<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1510</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>581</width>
     <height>501</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>A</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>B</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>C</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>D</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>E</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>F</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>G</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>H</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>I</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>J</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>190</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>250</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>330</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>250</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>250</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>turn over</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>20</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Не выбран корабль</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Шлюпка №1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Шлюпка №2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Шлюпка №3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Шлюпка №4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Эсминец №1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Эсминец №2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Эсминец №3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Крейсер №1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Крейсер №2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Линкор №1</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>490</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>490</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
